services/harmony/test/service-image-tags.ts

```diff
@@ -580,4 +580,186 @@ describe('Service image endpoint', async function () {
       });
     });
   });
+
+  describe('Enable and disable service image tag update', async function () {
+    describe('when a user is not in the EDL service admin groups', async function () {
```

Comment:
I think we should also verify that if the user is in the deployers group they cannot enable/disable (since other routes similar to this one are allowed for service deployers).